OsayamenOsaretin/post-it-app
# Add a tour for new users

#### What does this PR do?
This PR adds a minimalist onboarding solution to the App.

#### Description of task to be completed
Use onboarding solution to show users how to use the app.

#### How should this be manually tested?
After cloning, run app `npm run start-dev`.
- navigate to http://localhost:6969
- Sign in to the app and you should see brief descriptions of parts of the app and what they do

#### Screenshots
<img width="1440" alt="screen shot 2017-11-20 at 5 04 53 pm" src="https://user-images.githubusercontent.com/16901595/33028032-00954b8e-ce15-11e7-8c4e-a228ea7dcfd4.png">
